scripts/commands.js

```diff
@@ -2,27 +2,29 @@ require('dotenv').config()
 const { BOT_PREFIX } = process.env
 const Discord = require('discord.js')
 
+const CHANNELS = {
+    'stopStudyChannel': '882783531531653210'
+}
+
 const changeEnv = require('../server.js')
 
 exports.study = async (message, args, guildRoles) => {
     if(!typeof([args][0], 'int') && args) return message.reply(`To enter study mode:\n${BOT_PREFIX}study`)
-    args
-    ?
-    message.author.send(`You\'re now studying for ${args} minutes`)
-    :
+    
     message.author.send('You\'re now studying!')
 
-    message.guild.channels.cache.get('882783531531653210').send(`Welcome ${message.author} to Study Mode. Type \`${BOT_PREFIX}stop\` to end the session.`)
+    message.guild.channels.cache.get(CHANNELS.stopStudyChannel).send(`Welcome ${message.author} to Study Mode. Type \`${BOT_PREFIX}stop\` to end the session.`)
     guildRoles.forEach(r => {
-        r.role.name === 'Studying' ? message.member.roles.add(r.role.id) : ''
+        if(r.role.name === 'Studying') message.member.roles.add(r.role.id)
     })
 
     await message.delete()
 }
 
 exports.stop = async (message, args, guildRoles) => {
+    if (message.channelId !== CHANNELS.stopStudyChannel) return 0
```

Comment:
Why return 0? Just use a bare return statement - you don't use this return value

Reply:
I normally return 0 if something isn't correct and 1 if it is perfectly fine. Would it be more usual to see a bare return?

Reply:
If you were checking this return value, then it would be sensible, but most of the commands don't return anything at all - so if you're going to make them return something if all is normal, then all of them need to do it

Also side-note, from my experience, people use return 0 to signify everything being correct, and every other number is usually interpreted as an error code. For example, a quote from stackoverflow talking about C: 
"The return value for main indicates how the program exited. Normal exit is represented by a 0 return value from main. Abnormal exit is signaled by a non-zero return"

Reply:
Oh. I'll fix that and keep it in mind for future projects.